letsdrill.rb

```diff
@@ -1,16 +1,20 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
-
+	if integer >= 80 && integer <90 
+		"B"
```

Comment:
These work, but there's a shorter way to assess a range of numbers in Ruby. See if you can find out how to do that to shorten up these comparisons.

Also, an If/Else structure works, but it's not exactly the most efficient means of finding the letter grade you need. Should the grade be an F, it'll have to run through all the other assessments before getting to the one that's right. Do a little research on other ways to handle conditional structures to see if you can find one that will both make this faster, and reduce the amount of code needed.